react/tic-tac-toe/src/index.js

```diff
@@ -2,32 +2,42 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { AppContainer } from 'react-hot-loader';
 import { Provider } from 'react-redux';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+import { ConnectedRouter } from 'connected-react-router';
 
 import App from './app';
+import LoginForm from './app/screens/Login';
 import './config/i18n';
 import './scss/application.scss';
 import { register } from './serviceWorker';
-import store from './redux/store';
+import store, { history } from './redux/store';
 
 const render = () => {
   ReactDOM.render(
-    <Provider store={store}>
-      <AppContainer>
-        <App />
-      </AppContainer>
-    </Provider>,
+    <AppContainer>
+      <Provider store={store}>
+        <ConnectedRouter history={history}>
+          <Router>
+            <div>
```

Comment:
This `div` is need ?

Reply:
These corrections are made at https://github.com/wolox-training/oc-react/pull/21